gengokit/httptransport/httptransport.go

```diff
@@ -101,6 +111,24 @@ func NewBinding(i int, meth *deftree.ServiceMethod) *Binding {
 		nField.LowCamelName = LowCamelName(nField.Name)
 
 		nBinding.Fields = append(nBinding.Fields, &nField)
+
+		// Emit warnings for certain cases
+		if nField.IsBaseType == false {
```

Comment:
I'm unaccustomed to seeing boolean checks with either `== true` or `== false`. It's redundant in the true case. I suppose it's a bit more explicit in the negated case, in that it's harder to miss than a `!` bang, but something still strikes me odd about explicit checks against `true` or `false`.